pinot-transport/src/main/java/com/linkedin/pinot/transport/scattergather/ScatterGatherImpl.java

```diff
@@ -477,7 +478,7 @@ private void runInternal() {
       } finally {
         _requestDispatchLatch.countDown();
         BrokerRequest brokerRequest = (BrokerRequest) _request.getBrokerRequest();
-        _brokerMetrics.addMeteredQueryValue(brokerRequest, BrokerMeter.REQUEST_CONNECTION_WAIT_TIME_IN_MILLIS, timeWaitedMillis);
+        _brokerMetrics.addPhaseTiming(brokerRequest, BrokerQueryPhase.REQUEST_CONNECTION_WAIT, timeRemainingMillis*1000000);
```

Comment:
This changed from `timeWaitedMillis` to `timeRemainingMillis`? Also, is it possible to use actual nanoseconds instead of milliseconds, millisecond granularity is pretty coarse.